resources/css/components/category.css

```diff
@@ -0,0 +1,40 @@
+.pop-category {
```

Comment:
I think something along ".listing-category" or ".filter-category"/".category-filter" would be better naming, considering POP is Dutch